src/main/java/util/ApplicationUtils.java

```diff
@@ -0,0 +1,22 @@
+package util;
+
+import org.commcare.api.persistence.SqlSandboxUtils;
+
+/**
+ * Utility methods for dealing with Applications
+ */
+public class ApplicationUtils {
```

Comment:
punted on moving this for now since FileUtils is only defined in the `test` directory